ui/js/models/resultsets_store.js

```diff
@@ -301,8 +301,7 @@ treeherder.factory('ThResultSetStore', [
                     shownJobs.push(jMap.job_obj);
                 }
                 if (_.size(shownJobs) === spaceRemaining) {
-                    thNotify.send("Max pinboard size of " + maxSize + " reached.",
-                                  "danger");
+                    thNotify.send(errorMessage, "danger");
```

Comment:
Again these are the scenarios in the PR where things like `danger` could be single-quoted since it is an internal string denoting a bootstrap style and not an external string the user sees. But I defer to you guys :)